server/forge/common/status.go

```diff
@@ -58,21 +59,21 @@ func GetPipelineStatusContext(repo *model.Repo, pipeline *model.Pipeline, workfl
 func GetPipelineStatusDescription(status model.StatusValue) string {
 	switch status {
 	case model.StatusPending:
-		return "Pipeline is pending"
```

Comment:
this is about the description please revert it back!

if it is used somewhere, and that is to long, the StatusValue should be used directly

Reply:
What else could be here?
[Screenshot 2024-08-31-1](https://github.com/user-attachments/assets/c097de93-6f2a-4c0f-878d-070168a8a4d5)
[Screenshot 2024-08-31-2](https://github.com/user-attachments/assets/36790461-33b7-4b68-911e-eaab6a4f31f8)
[Screenshot 2024-08-31-3](https://github.com/user-attachments/assets/72d97e09-163b-4ff5-ad2f-aa41fb1199c0)

Reply:
Pipeline or workflow, BTW? :)

Reply:
ah well yes it should be renamed to workflow :D